server/chat-plugins/username-prefixes.ts

```diff
@@ -109,11 +154,13 @@ export const commands: Chat.ChatCommands = {
 
 			const types = target ? [prefixManager.validateType(toID(target))] : ['privacy', 'modchat'];
 
+			const entries = Config.forcedprefixes.filter((x: any) => types.includes(x.type)).map((x: any) => x.prefix);
+
 			return this.sendReplyBox(types.map(type => {
-				const info = Config.forcedprefixes[type].length ?
-					`<code>${Config.forcedprefixes[type].join('</code>, <code>')}</code>` : `none`;
+				const info = entries.filter((x: any) => x.type === type).length ?
+					`<code>${entries.filter((x: any) => x.type === type).join('</code>, <code>')}</code>` : `none`;
```

Comment:
`entries` should be an array of strings, right, since you've mapped `x => x.prefix`?

Reply:
oh yeah it is, ill fix in a bit